x/gamm/pool-models/balancer/export_test.go

```diff
@@ -0,0 +1,6 @@
+package balancer
+
+var (
```

Comment:
Why are we aliasing here making private one public?

Reply:
To be able to test unexported functions in tests. Files with `_test.go` do not get compiled into the final binary. As a result, these aliases are only accessible in tests